packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -284,6 +284,10 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
     });
   }
 
+  async cancelMessageEdit(/*messageId: string*/): Promise<void> {
```

Comment:
Is this function called by any composite code or impact our own state? If not, I would say it is currently no need to expose it here

The adapter is a layer designed 100% fit for composite, it satisfies and only satisfies what is needed by composite - if someone wants to do more, they can involke stateful client with usePropsFor or directly use stateful client

cancelMessageEdit is currently a pure UI thing, no impacts on either adapter states or client states, so it is not something we would need to adapter layer, at least for now

Reply:
Sure, we can skip it then; the reason for this PR was a code review item from the original PR. 

It might be nices to be able to see how to implement the functionality if needed.